### PR TITLE
[WIP] Install edits for 3.10

### PR DESCRIPTION
**Pull request by mburke5678**

### CODE REVIEW

**Comment by vikram-redhat** [comment]
@mburke5678  - what bug are these changes for?

**Comment by vikram-redhat** [comment]
@mburke5678 - what issue or bug are these changes for?

File: install/host_preparation.adoc [hunk 1]
```diff
@@ -40,8 +40,12 @@ The `PATH` for the root user on each host must contain the following directories
 - *_/usr/bin_*
 - *_/usr/sbin_*
 
-These directories set by default in a new RHEL 7.x installation.
+These directories set by default in a new RHEL 7.x installation. Run the following command to verify your `PATH` settings:
 
+----
+echo $PATH
```
Comment (kalexand-rh) [inline install/host_preparation.adoc]: \#

Make the addition a new paragraph, and I'd say "To verify your \`PATH\` settings, run the following command:"

File: install/prerequisites.adoc [hunk 2]
```diff
@@ -335,6 +335,16 @@ failure with:
 - Access to the {product-title} web console, because it is not accessible via
 IP address alone
 
+//from https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/7/html/networking_guide/sec-installing_networkmanager
+NetworkManager is installed by default on Red Hat Enterprise Linux. If necessary, to ensure that it is, enter the following command:
```
Comment (kalexand-rh) [inline install/prerequisites.adoc]: If it's installed by default, why do we need to check?

File: install/prerequisites.adoc [hunk 3]
```diff
@@ -379,6 +389,15 @@ $ dig master.example.com @10.64.33.1 +short
 $ dig node1.example.com @10.64.33.1 +short
 10.64.33.101
 ----
++
+====
+If the *dig* command is not found, use the following command to install the package,
```
Comment (kalexand-rh) [inline install/prerequisites.adoc]: Please add "bind-utils" to the list of prereq packages instead.

File: install/configuring_inventory_file.adoc [hunk 4]
```diff
@@ -805,7 +805,7 @@ the
 xref:prerequisites.adoc#system-requirements[latest
 installation documentation]. A user-defined value for minimum memory
 requirements might be set by setting the `openshift_check_min_host_memory_gb`
-cluster variable in your inventory file.
+cluster variable in the *[OSEv3:vars]* section of your inventory file.
```
Comment (kalexand-rh) [inline install/configuring_inventory_file.adoc]: Don't all the cluster variables go in that section?

File: install/host_preparation.adoc [hunk 5]
```diff
@@ -168,6 +172,13 @@ Alternatively, disable all repositories:
  yum-config-manager --disable \*
```
Comment (kalexand-rh) [inline install/host_preparation.adoc]: \#

File: install/host_preparation.adoc [hunk 6]
```diff
@@ -168,6 +172,13 @@ Alternatively, disable all repositories:
...
 ----
 +
+[NOTE]
```
Comment (kalexand-rh) [inline install/host_preparation.adoc]: I think it would be more helpful to add a separate step to install the yum-config-manager package before this step.

**Review by kalexand-rh — COMMENTED** [review]
@mburke5678, I have some comments and requests. PTAL?

**Comment by kalexand-rh** [comment]
@vikram-redhat, @mburke5678 ran into a snag when he was trying to install an environment to do some testing. I don't think there's a bug or issue for it.

**Comment by vikram-redhat** [comment]
@mburke5678 is this PR still valid?

**Comment by vikram-redhat** [comment]
@mburke5678 I would drop it considering all our other work.

**Pull request closed without merging**